src/app/GitUI/CommandsDialogs/RevisionDiffControl.cs

```diff
@@ -346,6 +346,7 @@ public void InitSplitterManager(SplitterManager splitterManager)
         {
             splitterManager.AddSplitter(DiffSplitContainer, nameof(DiffSplitContainer));
             splitterManager.AddSplitter(LeftSplitContainer, $"{nameof(RevisionDiffControl)}.{nameof(LeftSplitContainer)}");
+            BlameControl.InitSplitterManager(splitterManager, parentContainerPrefix: Name);
```

Comment:
`parentContainerPrefix: Name` may work for this PR. But it is not exactly what is meant. The _prefix_ shall contain the trailing dot - in order to apply this as generic pattern in a follow-up PR.